File: invesdwin-context-parent/invesdwin-context-scala/src/test/java/de/invesdwin/context/scala/tests/InputsAndResultsTestPercent.scala
System.out.println("getPercent")
if(getPercent != null)
	throw new Exception("getPercent already defined!")
val getPercent = putPercent
val getPercentType = getPercent.getClass()
System.out.println(getPercentType)
System.out.println(getPercent)
if(getPercentType != classOf[java.lang.Double])
	throw new Exception("getPercent not Double!")

System.out.println("getPercentVector")
if(getPercentVector != null)
	throw new Exception("getPercentVector already defined!")
val getPercentVector = putPercentVector.asInstanceOf[Array[Double]]
val getPercentVectorType = getPercentVector(0).getClass()
System.out.println(getPercentVectorType)
System.out.println(getPercentVector)
if(getPercentVectorType != classOf[Double])
	throw new Exception("getPercentVector not Double!")

System.out.println("getPercentVectorAsList")
if(getPercentVectorAsList != null)
	throw new Exception("getPercentVectorAsList already defined!")
val getPercentVectorAsList = putPercentVectorAsList.asInstanceOf[Array[Double]]
val getPercentVectorAsListType = getPercentVectorAsList(0).getClass()
System.out.println(getPercentVectorAsListType)
System.out.println(getPercentVectorAsList)
if(getPercentVectorAsListType != classOf[Double])
	throw new Exception("getPercentVectorAsList not Double!")

System.out.println("getPercentMatrix")
if(getPercentMatrix != null)
	throw new Exception("getPercentMatrix already defined!")
val getPercentMatrix = putPercentMatrix.asInstanceOf[Array[Array[Double]]]
val getPercentMatrixType = getPercentMatrix(0)(0).getClass()
System.out.println(getPercentMatrixType)
System.out.println(getPercentMatrix)
if(getPercentMatrixType != classOf[Double])
	throw new Exception("getPercentMatrix not Double!")

System.out.println("getPercentMatrixAsList")
if(getPercentMatrixAsList != null)
	throw new Exception("getPercentMatrixAsList already defined!")
val getPercentMatrixAsList = putPercentMatrixAsList.asInstanceOf[Array[Array[Double]]]
val getPercentMatrixAsListType = getPercentMatrixAsList(0)(0).getClass()
System.out.println(getPercentMatrixAsListType)
System.out.println(getPercentMatrixAsList)
if(getPercentMatrixAsListType != classOf[Double])
	throw new Exception("getPercentMatrixAsList not Double!")

File: invesdwin-context-parent/invesdwin-context-jruby/src/test/java/de/invesdwin/context/jruby/tests/InputsAndResultsTestPercent.rb
print("getPercent")
if (defined?(getPercent)).nil?
	throw "getPercent already defined!"
getPercent = putPercent
getPercentType = typeof getPercent
print(getPercentType)
print(getPercent)
if(getPercentType !== 'number')
	throw "getPercent not number!"

print("getPercentVector")
if (defined?(getPercentVector)).nil?
	throw "getPercentVector already defined!"
getPercentVector = putPercentVector
getPercentVectorType = typeof getPercentVector[0]
print(getPercentVectorType)
print(getPercentVector)
if(getPercentVectorType !== 'number')
	throw "getPercentVector not number!"

print("getPercentVectorAsList")
if (defined?(getPercentVectorAsList)).nil?
	throw "getPercentVectorAsList already defined!"
getPercentVectorAsList = putPercentVectorAsList
getPercentVectorAsListType = typeof getPercentVectorAsList[0]
print(getPercentVectorAsListType)
print(getPercentVectorAsList)
if(getPercentVectorAsListType !== 'number')
	throw "getPercentVectorAsList not number!"

print("getPercentMatrix")
if (defined?(getPercentMatrix)).nil?
	throw "getPercentMatrix already defined!"
getPercentMatrix = putPercentMatrix
getPercentMatrixType = typeof getPercentMatrix[0][0]
print(getPercentMatrixType)
print(getPercentMatrix)
if(getPercentMatrixType !== 'number')
	throw "getPercentMatrix not number!"

print("getPercentMatrixAsList")
if (defined?(getPercentMatrixAsList)).nil?
	throw "getPercentMatrixAsList already defined!"
getPercentMatrixAsList = putPercentMatrixAsList
getPercentMatrixAsListType = typeof getPercentMatrixAsList[0][0]
print(getPercentMatrixAsListType)
print(getPercentMatrixAsList)
if(getPercentMatrixAsListType !== 'number')
	throw "getPercentMatrixAsList not number!"
